pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/kube_ingress_reconciler.go

```diff
@@ -337,13 +343,17 @@ func (r *RawIngressReconciler) Reconcile(isvc *v1beta1.InferenceService) error {
 		return err
 	}
 	internalHost := getRawServiceHost(isvc, r.client)
+	var scheme string
 	if authEnabled {
 		internalHost += ":" + strconv.Itoa(constants.OauthProxyPort)
+		scheme = "https"
+	} else {
+		scheme = "http"
```

Comment:
same here:

```go
        scheme = "http"
	if authEnabled {
		internalHost += ":" + strconv.Itoa(constants.OauthProxyPort)
		scheme = "https"
	} 
```

considering, the logic you wrote is the correct behavior for both cases.

Reply:
addressed in latest commit